internal/seed/config/populate.go

```diff
@@ -97,28 +97,32 @@ func ImportConfiguration(root string, profile string) error {
 
 		LoggingClient.Debug("reading toml " + path)
 		// load the ToML file
-		config, _ := toml.LoadFile(path)
+		config, err := toml.LoadFile(path)
+		if err != nil {
+			LoggingClient.Warn(err.Error())
```

Comment:
Is there a reason we don't also propagate this error here with `return err`?

Reply:
Oh also, if you do return the error here, you should drop the else block and re-outdent the else block (in accordance with golint)

Reply:
If I propagate the error then the loop stops. You said in your comment you wanted the rest of the configs to be populated.

Reply:
Ah yes you're right sorry I thought ImportConfiguration was run in a loop somewhere, but I see that it runs the loop itself.